SpatialGDK/Source/SpatialGDKTests/SpatialGDK/Interop/SpatialWorkerFlags/SpatialWorkerFlagsTest.cpp

```diff
@@ -51,39 +51,160 @@ SPATIALWORKERFLAGS_TEST(GIVEN_a_flagUpdate_op_WHEN_removing_a_worker_flag_THEN_f
 	return true;
 }
 
-SPATIALWORKERFLAGS_TEST(GIVEN_a_bound_delegate_WHEN_a_worker_flag_updates_THEN_bound_function_invoked)
+// delegates
```

Comment:
What's the purpose of these comments?

Reply:
Grouping the tests a bit since we have only level of hierarchy to our tests I kind of got lost which related to which as I was updating / adding. Can remove if needed.